source/style-guide/style/parameters.txt

```diff
@@ -20,25 +18,13 @@ When documenting parameters, use the following guidelines:
 
    -  Write the first sentence of a description with an implied
       subject. For example, if the parameter is ``name``, the
-      description might be "Server name, which becomes the initial host
-      name of the server."
+      description might be "Initial host name of the server."
 
    -  Include any valid values and default value at the end of the
       description. Use the formats "Valid values are *n* and *n*." and
       "The default is *n*." For example, "Valid values are ``true`` and
       ``false``." and "The default is ``false``."
```

Comment:
**suggestion:** For body params, I have used:

```suggestion
   -  Include any valid values and default value at the end of the
      description. Use the formats "Application accepts *m* and *n*." and
      "The default is *n*." For example, "Atlas accepts ``true`` and
      ``false``." and "The default is ``false``."
```

For responses, I use:
```suggestion
   -  Include any valid values and default value at the end of the
      description. Use the formats "Application returns *m* or *n*." and
      "The default is *n*." For example, "Atlas returns ``true`` or
      ``false``." and "The default is ``false``."
```

The reason is to remove ambiguity as to whether something is probable, possible, or acceptable.

Reply:
This is a fair point, but also makes things harder for content reuse.

Reply:
**suggestion:** Yes, but that's a problem for later. Right now, we have an issue with trying to figure out "acceptable" vs. "possible". Making it active voice fixes that. If this is a living doc, make this change and we can revisit post-OpenAPI.

Reply:
This may be a moot point with OpenAPI, but it has been hard to determine the word with the best connotation: "acceptable"? "valid"? "possible"? All of those sound like they're part of what's acceptable.

Reply:
Addressed this in PR #64